### PR TITLE
test: Retry realmd operation if another is still active

### PR DESCRIPTION
**Pull request by mvollmer**

### CODE REVIEW

**Comment by mvollmer** [comment]
Hmm, this should probably count as a bug in realmd and we might simply catch it with a naughty...

File: test/verify/check-realms [hunk 1]
```diff
@@ -256,18 +256,29 @@ class TestRealms(MachineCase):
...
-        b.wait_text_not(".realms-op-message", "")
-        b.wait_not_visible(".realms-op-leave-only-row")
-        b.click(".realms-op-cancel")
-        b.wait_popdown("realms-op")
+        # Sometimes with some versions of realmd the Leave operation
+        # from above is still active in the realmd daemon.  So we loop
+        # here until we get the expected error instead of "Already
+        # running another action".
+
+        while True:
```
Comment (martinpitt) [inline test/verify/check-realms]: Right, that sounds like a bug, let's please at least report it in bz. Also, please let's not have an indefinite loop here, but at most one retry?

Comment (mvollmer) [reply test/verify/check-realms]: > let's please at least report it in bz

It only seems to happen on the Debians, and I am not familiar with reporting bugs there... *hint* *hint*

Comment (mvollmer) [reply test/verify/check-realms]: > Also, please let's not have an indefinite loop here, but at most one retry?

Done.

**Pull request merged**